app/jobs/create_account_fcrepo_job.rb

```diff
@@ -0,0 +1,6 @@
+# This job is the same as its parent, but chains to the next step
+class CreateAccountFcrepoJob < CreateFcrepoEndpointJob
```

Comment:
So that they share the same `#perform` implementation and reduce maintenance burden.  If we proliferated a number of different job chains, the value in keeping the extensions as light as possible is pronounced.  

The idea is to keep the job itself separate from its position in various sequences.  With this approach, a hypothetical rake task or admin button to `perform_now` could utilize the parent class and avoid chaining.  

If we decide we will never make use of such functionality, then no subclassing is needed.  But for purposes of demonstrating the minimal application of `after_perform`, this is ideal.